packages/salesforce/test/Adaptor.test.js

```diff
@@ -42,60 +26,7 @@ describe('Adaptor', () => {
         .then(state => {
           expect(spy.args[0]).to.eql([sObject, fields]);
           expect(spy.called).to.eql(true);
-          expect(state.references[0]).to.eql({ Id: 10 });
```

Comment:
I will investigate this further, it seems like i am getting undefined when i use `state.references[0]`

Reply:
Hiya @josephjclark since we are using `composeNextState` the data will be `state.data`. If we want to test `state.references[0]` that mean we have to run another operation so that `state.data` gets moved to `state.references`

Reply:
Ah gotcha, that makes sense. Thanks for checking this out!